hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerConfiguration.java

```diff
@@ -44,6 +44,7 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Timeout;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Heads up: `assertThat` is being added to `TestOzoneManagerConfiguration` and `TestOzoneManagerHAWithAllRunning` in #5844, too.  We'll need another round to resolve conflicts when that one is merged.

Reply:
ok, will roolback `TestOzoneManagerConfiguration` and `TestOzoneManagerHAWithAllRunning`